tools/release-readiness/polarion/export_comp_data_to_sheets.py

```diff
@@ -0,0 +1,343 @@
+            self.update_and_check(ws, "A2", "Tier Classification")
+            ws.format("A2", self.format_cell())
+            self.update_and_check(ws, "A3", "Day")
+            ws.format("A3", self.format_cell())
+            self.update_and_check(ws, "B3", "Label")
+            ws.format("B3", self.format_cell())
+            start_c = "C"
+
+            for component_filter_key in config.component_filter.keys():
+                curr_c = start_c
+                print("\n Comp keys:", component_filter_key)
+                for val in config.component_filter[component_filter_key]["values"]:
+                    # Pausing for accomodating the no. of writes per minute.
+                    time.sleep(5)
+                    print("Component: ", val)
+                    print("Current C value:", curr_c)
+
+                    for tier_val in config.tags:
+                        print("Tier val:", tier_val)
+                        self.update_and_check(ws, curr_c + "2", tier_val)
+                        curr_c = self.get_next_col(curr_c)
+                        print("current 'c' inside loop:", curr_c)
+
+                    print("Started merging of cells")
+                    ws.merge_cells(start_c + "1:" + self.get_prev_col(curr_c) + "1")
+                    self.update_and_check(ws, start_c + "1", val)
+                    ws.merge_cells(start_c + "3:" + self.get_prev_col(curr_c) + "3")
+                    self.update_and_check(ws, start_c + "3", "Automated")
+                    ws.format(
+                        start_c + "1:" + curr_c + "3",
+                        self.format_cell(bg=config.comp_color[val]),
+                    )
+                    start_c = self.get_next_col(curr_c)
+                    curr_c = self.get_next_col(curr_c)
+
+        row_count = self.next_available_row(ws)
+        day = str(datetime.datetime.now()).split(".")[0]
+        label = (
+            str(datetime.datetime.now().month) + "/" + str(datetime.datetime.now().day)
+        )
+        print("\nRow count :", row_count)
+        self.update_and_check(ws, "A" + row_count, day)
+
+        self.update_and_check(ws, "B" + row_count, label)
+
+        start_c = "C"
+        for comp in data.keys():
+            print("#######################")
+            print("Component :", comp)
+            for val in data[comp]:
+                print("Value in comp:", val)
+                self.update_and_check(ws, start_c + row_count, val)
+                start_c = self.get_next_col(start_c)
+                print("start_c inner for loop:", start_c)
+            start_c = self.get_next_col(start_c)
+            print("start_c outer for loop:", start_c)
+        start_c = self.get_prev_col(start_c)
+        print("start_c outside loop:", start_c)
+        ws.format("A" + row_count + ":" + start_c + row_count, self.format_cell())
+
+        """
+        #Charts part addition needed.
+        if row_count == "3":
```

Comment:
can we take this doc string out, if not needed.

Reply:
done